_includes/header.html

```diff
@@ -2,7 +2,8 @@
     <ul>
         <li><a href="{{site.baseurl}}/#services">Services</a></li>
         <li><a href="{{site.baseurl}}/#features">Features</a></li>
-        <li><a href="{{site.baseurl}}/#docs">Documentation</a></li>
+        <li><a href="{{site.baseurl}}/#docs">Getting Started</a></li>
+        <li><a href="http://docs.mathjax.org/">Documentation</a></li>
```

Comment:
If you change this to `https` you will not get the security warnings that you mentioned f2f.

There are a bunch of other links to the docs that are also `http:` in `_includes/frontpage/docs.html`.  Also, all the sponsor links seem to be `http:`, so perhaps those should go to `https:` as well.

Reply:
The issue here is that the `docs.mathjax.org` site is not secure. I can fix up all links to `https` but they might fall back to `http`.

Reply:
Correction: `https://docs.mathjax.org` actually works, but for some reason the link from `mathjax.org` still falls back to `http`. Might be my jekyll setup, though.